stress/shell/src/main/java/alluxio/stress/cli/StressMasterBench.java

```diff
@@ -116,7 +126,10 @@ public void prepare() throws Exception {
       Configuration hdfsConf = new Configuration();
       // force delete, create dirs through to UFS
       hdfsConf.set(PropertyKey.Name.USER_FILE_DELETE_UNCHECKED, "true");
-      hdfsConf.set(PropertyKey.Name.USER_FILE_WRITE_TYPE_DEFAULT, "CACHE_THROUGH");
+      if (!mParameters.mWriteType.isEmpty())
+      {
```

Comment:
this is incompatible with Alluxio java style

https://docs.alluxio.io/os/user/stable/en/contributor/Code-Conventions.html#javadoc-style